SofaKernel/modules/SofaHelper/src/sofa/helper/ComponentChange.cpp

```diff
@@ -473,6 +473,16 @@ std::map<std::string, ComponentChange> uncreatableComponents = {
     // SofaEigen2Solver was pluginized in #1635
     { "SVDLinearSolver", Pluginized("v20.12", "SofaEigen2Solver") },
 
+    // SofaBaseUtils was packaged in #1640
+    //{ "AddResourceRepository", Pluginized("v20.12", "SofaBaseUtils") },
+    //{ "AddPluginRepository", Pluginized("v20.12", "SofaBaseUtils") },
+    //{ "InfoComponent", Pluginized("v20.12", "SofaBaseUtils") },
+    //{ "MakeAliasComponent", Pluginized("v20.12", "SofaBaseUtils") },
+    //{ "MakeDataAliasComponent", Pluginized("v20.12", "SofaBaseUtils") },
+    //{ "MessageHandlerComponent", Pluginized("v20.12", "SofaBaseUtils") },
+    //{ "FileMessageHandlerComponent", Pluginized("v20.12", "SofaBaseUtils") },
+    //{ "RequiredPlugin", Pluginized("v20.12", "SofaBaseUtils") },
```

Comment:
why commented? 
most of them are not component if I'm right.

Reply:
I would say that the one with a Component in their name are :)

Reply:
Actually I took the decision to write those lines but commented them because:
 - SofaBaseUtils is actually pluginized so it **could** loaded optionnally
 - but is is forcefully loaded anyway in runSofa so the messages will never appear
 Basically I just let them here to keep a trace

Reply:
And of all of those are registered components (i.e registerObject is called on those classes, so added in the factory etc)